package.json

```diff
@@ -1,22 +1,52 @@
 {
-  "name": "progrrammers-only",
+  "name": "programmers-only",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start": "node dist/index.js",
+    "build": "tsc",
+    "lint": "eslint src/**/*.[jt]s",
+    "lint:fix": "eslint src/**/*.[jt]s --fix",
     "test": "jest",
     "test:watch": "jest --watch"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
+  "huky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,ts}": [
+      "npm run lint",
+      "npm run test"
+    ]
+  },
   "dependencies": {
-    "body-parser": "^1.19.0",
+    "@types/express": "^4.17.7",
+    "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mongoose": "^5.9.28"
+    "mongoose": "^5.9.28",
+    "passport": "^0.4.1",
+    "passport-github": "^1.1.0"
   },
   "devDependencies": {
+    "@types/passport": "^1.0.4",
```

Comment:
To better organisation, you should use some reordering eslint rules, it should be alphabetic order